azurerm/internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -1152,3 +1212,75 @@ func resourceAzureRMCosmosDBAccountVirtualNetworkRuleHash(v interface{}) int {
 
 	return schema.HashString(buf.String())
 }
+
+func expandCosmosdbAccountBackupPolicy(input []interface{}) (documentdb.BasicBackupPolicy, error) {
+	if len(input) == 0 || input[0] == nil {
+		return documentdb.BackupPolicy{
```

Comment:
Is there any reason to return a hypothetic type `BackupPolicy` here, rather than `nil`?